app/src/main/res/layout/activity_todo_test.xml

```diff
@@ -183,5 +183,17 @@
             app:fabSize="normal"
             android:layout_width="wrap_content"
             />
+
+        <android.support.design.widget.FloatingActionButton
+            android:id="@+id/doneToDoFloatingActionButton"
+            android:layout_marginEnd="16dp"
+            android:layout_marginRight="16dp"
```

Comment:
Could you extract values like `16dp` to `dimens.xml` or use existing there values names? Code would be then much easier to maintain